org.manasource.Mana.json

```diff
@@ -0,0 +1,76 @@
+{
+  "app-id": "org.manasource.Mana",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mana",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--persist=.config/mana",
+    "--persist=.local/share/mana"
+  ],
+  "modules": [
+    "shared-modules/physfs/physfs.json",
+    {
+        "name": "SDL2_gfx",
+        "rm-configure": true,
+        "build-options": {
+            "arch": {
+                "aarch64": {
+                    "config-opts": [
+                        "--disable-mmx"
+                    ]
+                }
+            }
+        },
+        "cleanup": [
+            "/include",
+            "/lib/pkgconfig"
+        ],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "http://www.ferzkopp.net/Software/SDL2_gfx/SDL2_gfx-1.0.4.zip",
+                "sha256": "b6da07583b7fb8f4d8cee97cac9176b97a287f56a8112e22f38183ecf47b9dcb"
+            },
+            {
+                "type": "script",
+                "dest-filename": "autogen.sh",
+                "commands": [
+                    "autoreconf -vfi"
+                ]
+            }
+        ]
+      },
+      {
+          "name": "Guichan",
```

Comment:
this module need the autoconf file updated.

See https://github.com/flathub/org.gimp.GIMP/blob/master/org.gimp.GIMP.json#L282 for an example. (the `shell` source)

Reply:
Thanks! Would alternatively the same approach used for SDL2_gfx above work as well?

Reply:
no. not the same issue